db/seeds/development/serials.seeds.rb

```diff
@@ -3,7 +3,7 @@
 
   posts = site1.posts.published.order_by_recently.cycle
 
-  5.times do |i|
+  26.times do |i|
```

Comment:
`26` is a magic number. I don't understand what this number means.
Probably you want to show the second page of serials?
If you want to do that, you should add comment about this number.